bundles/org.openhab.binding.zway/src/main/java/org/openhab/binding/zway/internal/converter/ZWayDeviceStateConverter.java

```diff
@@ -109,13 +112,9 @@ private static State getPercentState(String multilevelValue) {
      * @param binary switch value
      * @return transformed openHAB state
      */
-    private static State getBinaryState(String binarySwitchState) {
+    private static State getBinaryState(@Nullable String binarySwitchState) {
         if (binarySwitchState != null) {
-            if (binarySwitchState.equals("on")) {
-                return OnOffType.ON;
-            } else if (binarySwitchState.equals("off")) {
-                return OnOffType.OFF;
-            }
+            return OnOffType.from(binarySwitchState);
```

Comment:
This will return `OFF` when neither "on" or "off", so it breaks the previous behavior where the method would ultimately return `UNDEF`.

Reply:
Fixed